mscore/pianoroll/pianoview.cpp

```diff
@@ -510,6 +520,117 @@ void PianoView::drawBackground(QPainter* p, const QRectF& r)
       }
 
 
+//---------------------------------------------------------
+//   drawNoteBlock
+//---------------------------------------------------------
+
+void PianoView::drawNoteBlock(QPainter* p, PianoItem* block)
+{
```

Comment:
still a brace indent issue

Reply:
Should be fixed now.

Reply:
Not the one at the end of that method ;-)